### ISSUE TITLE
Unit Testing 

### ISSUE THREAD

**Issue by levlaz** [silverp1/space-personal#33]
We need to write some unit tests, do CI, all that jenky devops stuff. This will help us make a high quality product and catch weird errors when we release. I will work on this slowly, but if we do not start now it will be way too late later.

**Comment by jdppettit** [comment]
I agree - I did a little research on this topic but I couldn't really find a decent guide for writing the actual tests. Do you know of any?

**Comment by levlaz** [comment]
Someone recommended this: http://docs.python-guide.org/en/latest/writing/tests/ 

I wrote a bunch of unit tests in Java and C# before but never for python projects. I do think that this is very important we should start doing this asap. :) 

We should also get jenkins or travis to run some of these tests.
